.github/renovate.json

```diff
@@ -32,6 +32,11 @@
     "addLabels": ["automerge"]
   },
   "packageRules": [
+    {
+      "description": "Temporary rule to prevent exposure to CVE-2024-57699 in newer versions of json-smart.",
+      "matchPackageNames": ["net.minidev:json-smart"],
+      "allowedVersions": "<=2.4.11"
+    },
```

Comment:
no strong objection to including this, but do we also need an issue to follow-up on this? Essentially, how do we get to the latest json-smart version once it's available?

Reply:
This is a valid point. Including a temporary workaround for CVE-2024-57699 is helpful for now, but it’s equally important to track the resolution of this issue to eventually upgrade to the latest.
I will create a follow-up issue and add to the board in order to monitoring the library and remove the rule in the future.